### PR TITLE
Take the proper interval from database

### PR DESCRIPTION
**Pull request by jmbenlloch**
@andLaing pointed out to me that we are not taking the proper interval when querying the database as stated in #364. This PR fixes that issue.

### CODE REVIEW

**Comment by gonzaponte** [comment]
Can you add a short test or should we leave it for another PR?

**Comment by gonzaponte** [comment]
Closes #364.

**Comment by jmbenlloch** [comment]
I don't have time for that test now, the queries are somehow complex and I don't think a toy example would be enough.

**Review by gonzaponte — APPROVED** [review]
Important fix that has been in our to-do list for a long time. We are accumulating a bit of technical debt by not adding a test, but there is a hurry to introduce this PR.

**Comment by gonzaponte** [comment]
We should open an empty PR so we don't forget to add a test.

**Comment by jmbenlloch** [comment]
TLDR;

**IMPORTANT**: Without this update, the code is **NOT** working with the current DB. So everyone should update to this commit after merge and **must run** `bash manage.sh download_test_db` to get the corrected DB.

Extended version:

There was another error in the DB code. It was assuming the mapping for all PMTs had to start at the same run. We have recently change only one PMT getting this status in the DB:

```
+--------+--------+--------+----------+
| MinRun | MaxRun | ElecID | SensorID |
+--------+--------+--------+----------+
|   5025 | 100000 |      0 |        0 |
|   5025 | 100000 |     12 |        1 |
|   5025 | 100000 |      1 |        2 |
|   5025 | 100000 |     13 |        3 |
|   5025 | 100000 |     19 |        4 |
|   5025 | 100000 |     18 |        5 |
|   5025 | 100000 |      7 |        6 |
|   5025 | 100000 |      6 |        7 |
|   5025 |   5153 |     17 |        8 |
|   5025 | 100000 |     16 |        9 |
|   5025 | 100000 |      5 |       10 |
|   5025 | 100000 |      4 |       11 |
|   5154 | 100000 |      2 |        8 |
+--------+--------+--------+----------+
13 rows in set (0,00 sec)
```

And the code was only returning the latest one (minrun 5154, sensor id 8). I have fixed the query to get the proper ones and tests are passing again.

For it to work I have also fixed some old values from the DB in the table `ChannelGain`, in particular this two queries:
```
UPDATE ChannelGain SET MaxRun = 2062 where MinRun = 1755;
UPDATE ChannelGain SET MaxRun = 2842 where MinRun = 2063;
```

The problem with them was that `MaxRun` was set to 100000 for both of them, so if we tried to get the values for a current run `MinRun <= 5168 and 5168 <= MaxRun`, we would get the current value but also two old and incorrect ones. I have check the status of all relevant tables to make sure `MaxRun` was correctly set in all of them and so is the case now. `ChannelGain` was the only one with this problem.

To summarize, the code can use now `MinRun` and `MaxRun` in a correct way after fixing some values in the DB.

**Comment by gonzaponte** [comment]
Ok. PLEASE MERGE!

**Comment by jmbenlloch** [comment]
Done, already merged!

**Comment by jacg** [comment]
> everyone should update to this commit after merge and must run bash manage.sh download_test_db

Of course, this shouldn't need to be done manually at all! It should should happen automatically because the DB should be version controlled. How is the Git LFS work progressing?

**Comment by jjgomezcadenas** [comment]
No progress. Jose Vicente proposed an scheme, as you remember, but no one has had any time to look into this. Right now we are terribly undermanned, with the usual suspects horribly over-committed.

**Comment by jacg** [comment]
Is #210 the only issue or PR that we have specifically addressing this?
